kimchi/src/circuits/lookup/index.rs

```diff
@@ -64,6 +64,12 @@ pub struct LookupConstraintSystem<F: FftField> {
     #[serde_as(as = "Option<o1_utils::serialization::SerdeAs>")]
     pub runtime_selector: Option<E<F, D<F>>>,
 
+    /// Optional runtime tables, listed as tuples `(length, id)`.
+    pub runtime_tables: Option<Vec<RuntimeTableSpec>>,
+
+    /// The offset of the runtime table within the concatenated table
+    pub runtime_table_offset: Option<usize>,
```

Comment:
I think at this point it'd be interesting to create an `runtime_index: Option<RuntimeIndex>` to avoid all the options there

Reply:
good idea